unfoldingWord/translationCore
# Cw misc cleanup

#### This pull request addresses:

Issues #1574, #1529, #1566, #1579, plus some updating to global colors and removing old styling and replacing React Buttons with standard buttons using global styling classes.



#### How to test this pull request:

Go to import page, type a url, go to another page and come back, make sure URL is blank.
Now type a url and then hit refresh, make sure it's cleared.
Enter a url with whitespace in front of it, make sure it still works


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1653)
